app/controllers/missions_controller.rb

```diff
@@ -0,0 +1,47 @@
+class MissionsController < ApplicationController
+  def index
+    @missions = Mission.all
+  end
+
+  def new
+    @mission = Mission.new
+  end
+
+  def create
+    @mission = Mission.new(mission_params)
+
+    if @mission.save
+      redirect_to root_path, notice: "新增任務成功"
```

Comment:
字串除非有如 `"interpolate_#{some_variable}"` 等 interpolation 的必要，不然請維持單引號。 請做出對應的調整，將所有不需要用到雙引號包起來的字串改用單引號包起來。

Ref: https://github.com/rubocop/ruby-style-guide#consistent-string-literals-single-quote

```suggestion
      redirect_to root_path, notice: '新增任務成功'
```